crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -510,12 +510,14 @@ export namespace cst {
     get kind(): RuleKind;
     get textLength(): TextIndex;
     get children(): Array<RuleNode | TokenNode>;
+    get cursor(): Cursor;
   }
   export class TokenNode {
     get type(): NodeType.Token;
     get kind(): TokenKind;
     get textLength(): TextIndex;
     get text(): string;
+    get cursor(): Cursor;
```

Comment:
I suggest adding a unit test that checks/asserts the cursor results in TypeScript.

Reply:
I want to do that with the `Iter` work. But I will add a gumby test at least.